src/mlpack/tests/activation_functions_test.cpp

```diff
@@ -330,6 +330,54 @@ void CheckPReLUGradientCorrect(const arma::colvec input,
   BOOST_REQUIRE_CLOSE(gradient(0), target(0), 1e-3);
 }
 
+/*
+ * Implementation of the CReLU activation function test. The function
+ * is implemented as CReLU layer in the file c_relu.hpp
+ *
+ * @param input Input data used for evaluating the CReLU activation
+ *   function.
+ * @param target Target data used to evaluate the CReLU activation.
+ */
+void CheckCReLUActivationCorrect(const arma::colvec input,
```

Comment:
Do you think we can integrate this into the test itself, to reduce the number of lines?

Reply:
@zoq , I just followed the pattern , other test had similar pattern so i just followed it , But i agree you with , we could create only one object and do the required , I made apt changes.